tests/accidental_tests.ts

```diff
@@ -274,9 +272,6 @@ function basic(options: TestOptions): void {
   notes.forEach((note, index) => {
     Note.plotMetrics(f.getContext(), note, 140);
     options.assert.ok(note.getModifiersByType('Accidental').length > 0, 'Note ' + index + ' has accidentals');
-    note.getModifiersByType('Accidental').forEach((accid: Modifier, index: number) => {
-      options.assert.ok(accid.getWidth() > 0, 'Accidental ' + index + ' has set width');
-    });
```

Comment:
Just curious why this part of the test is no longer needed?

Reply:
Back in. I thought that it was not adding value to check that a glyph has a width.